gatsby-node.ts

```diff
@@ -387,6 +416,20 @@ export const onCreatePage: GatsbyNode<any, Context>["onCreatePage"] = async ({
   }
 }
 
+export const onPostBootstrap: GatsbyNode["onPostBootstrap"] = ({ actions }) => {
+  const { createRedirect } = actions
+
+  supportedLanguages.forEach((lang) => {
```

Comment:
Creating these 404 redirects once all the pages and other redirects were created. We need them at the end of the `_redirects` file.

Reply:
So if understand, the _redirects file that is created when building contains all redirects for the pages on the site. The site tries to resolve one of those initial redirects for the url, but if it can't find it then falls back to these 404 redirects being created here? Hopefully, this question makes sense. If it doesn't let me know.

Reply:
Good question.

>  the _redirects file that is created when building contains all redirects for the pages on the site

Yes. Including the 404s. These files (_redirects & redirects.json) are only created for servers. They don't play any role in the local env.

> The site tries to resolve one of those initial redirects for the url, but if it can't find it then falls back to these 404 redirects being created here?

Yes, I think the complete order is:
1. `url/path` match file => display the file
2. `url/path` match redirect => display the redirected file
3. `url/path` doesn't match anything => display default 404 (in English)
    - unless it matches the 404 fall backs for each locale (the ones created here), e.g. `/es/something`, will display the `/es/404` page